src/Symfony/Component/String/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/string",
+    "type": "library",
+    "description": "Symfony String component",
+    "keywords": ["string", "utf8", "utf-8", "grapheme", "i18n"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "symfony/polyfill-intl-grapheme": "~1.0",
+        "symfony/polyfill-intl-normalizer": "~1.0",
+        "symfony/polyfill-mbstring": "~1.0"
+    },
+    "autoload": {
+        "psr-4": { "Symfony\\Component\\String\\": "" },
+        "files": [ "Resources/functions.php" ],
+        "exclude-from-classmap": [
+            "/Tests/"
+        ]
+    },
+    "minimum-stability": "dev",
```

Comment:
As for all Symfony components: because we want to test with the next versions of our deps, to be sure they're on track and spot any issues as early as possible.